roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -224,6 +224,10 @@ container_manager: docker
 # When enabled, it requires `container_manager` different than Docker
 kata_containers_enabled: false
 
+# Enable crun as additional container runtime
+# When enabled, it requires `container_manager` different than Docker
```

Comment:
Require crio ? or is it available using containerd ?

Reply:
Good catch, this only works with CRI-O. I'll change it